content/blog/backstage-weekly-28-adopting-backstage/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Backstage Weekly 28 - A guide to adopting Backstage
+date: '2021-05-31T21:00:00.0Z'
+description: Plus... a recap of community session 4 and a look at the new Backstage catalog UI.
+author:
+  name: David Tuite
+tags: ['newsletter']
+---
+
+Having talked to many companies who are adopting Backstage, one thing is clear... Backstage means different things to different people. Because Backstage is a flexible and highly customizable framework, it can be tricky to figure out how to get started with it.
```

Comment:
```suggestion
Having talked to many companies who are adopting Backstage, one thing is clear... Backstage means different things to different people. Because Backstage is a flexible and highly customizable application, it can be tricky to figure out how to get started with it.
```

Framework seems wrong, I don't like application either though...

Reply:
@Xantier platform? :)

Reply:
I'll go with platform.